onward/app/feed/MostPopularAgent.scala

```diff
@@ -59,8 +61,13 @@ class GeoMostPopularAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi
         item: JsValue <- ophanResults.asOpt[JsArray].map(_.value).getOrElse(Nil)
         url <- (item \ "url").asOpt[String]
       } yield {
-        contentApiClient.getResponse(contentApiClient.item(urlToContentPath(url), Edition.defaultEdition))
-          .map(_.content.map(RelatedContentItem(_)))
+        contentApiClient
+          .getResponse(contentApiClient.item(urlToContentPath(url), edition).showTags("tone"))
```

Comment:
I think you need section, paid-content tags and isAppropriateForSponsorship field.